designer/settings/base.py

```diff
@@ -224,7 +224,7 @@
 EXTRA_SCOPE = ['permissions']
 
 # TODO Set this to another (non-staff, ideally) path.
-LOGIN_REDIRECT_URL = '/admin/'
```

Comment:
Do we still need the above TODO?

Reply:
That's from the cookiecutter. I'll toss it.